source/glest_game/world/scenario.cpp

```diff
@@ -237,9 +237,25 @@ namespace Glest {
       return isTutorial;
     }
 
-    void Scenario::loadScenarioInfo(string file,
+    bool Scenario::loadScenarioInfo(string file,
                                     ScenarioInfo * scenarioInfo,
                                     bool isTutorial) {
+      if (fileExists (file) == false)
+      {
+        // FIXME: A new string will need a new translation; we haven't
+        // discussed how we'll deal with translations yet.
+        //
+        // this particular string is just a placeholder, and will hopefully
+        // get changed soon, when we can point a user to complete documentation
+        // on how to obtain a scenario
```

Comment:
We will still be using the mod center, but not all scenarios will be put there. We want a way to make it easier for users to obtain scenarios that aren't included with the distribution.